coralproject/talk
# CSS Customization and i18n fixes

## What does this PR do?

- Custom CSS is now loaded on all admin pages, giving admins the ability to change the styling on those pages.
- Fixes and improvements were made to the mailing service.
- Existing styles for secondary admin pages (password reset, confirm email) were updated.

## How do I test this PR?

1. Create a fresh Talk install.
2. Install Talk.
3. Create a new account (with email enabled).
4. Click confirm email dialog, and follow steps, notice the styling.
5. Change the custom css url to something like bootstrap.
6. Repeat 1-4, but notice the new styles :).

## Pending tasks

- [ ] translate admin page ui elements to es
